dal/src/test/kotlin/com/home/generator/JooqGenerator.kt

```diff
@@ -0,0 +1,55 @@
+package com.home.generator
+
+import org.jooq.util.GenerationTool
+import org.jooq.util.jaxb.*
+import org.jooq.util.jaxb.Target
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+import java.io.File
+import java.io.FileInputStream
+import java.util.*
+
+class JooqGenerator {
+
+    @Test
```

Comment:
I would suggest to use an app class instead of test to generate JOOQ classes.

Reply:
Will be refactored as main class but I will keep sources in test scope